lib/api/controllers/collection.js

```diff
@@ -361,6 +366,31 @@ class CollectionController extends NativeController {
       .then(() => null);
   }
 
+  /**
+   * Refresh an internal collection
+   *
+   * @param {Request} request
+   * @returns {Promise}
+   */
+  async refreshInternal (request) {
```

Comment:
I understand you point of view but in this case we can also put this route in the `security` controller since the documents manipulated by this controllers are stored in internal collections

Reply:
Maybe we have a hard time to decide where to put it because it does too many things?

For instance, it might be better to have a `security:refresh` route which refreshes all security-related collections, and an `admin:refresh` one refreshing all internal collections whatsoever?

Reply:
As we discuss, I agree with the `security:refresh` action only on `users`, `profils` and `roles` collection.  
We will see if we need an `admin:refresh` action later